README.md

```diff
@@ -8,6 +8,13 @@ This repository houses the non-compiler components of the system.
 
 The compiler repo can be found [here](https://github.com/softdevteam/ykrustc).
 
+## Getting Started
+
+To get started, see
+[this section in the documentation](https://softdevteam.github.io/ykdocs/dev/getting_started.html).
+
+Note that you must use `cargo xtask` commands when dealing with this repo.
```

Comment:
Can we say something like "`yk` uses `cargo-xtask` instead of normal cargo. You will need to have [xtask](https://github.com/matklad/cargo-xtask/) installed (e.g. with `cargo install cargo-xtask`). Instead of `cargo x` (for some command `x`), you will then need to execute `cargo xtask x`." here?

Reply:
Try 121bcc5.

We don't have to install anything, it all works magically with a cargo alias.

Reply:
How does `xtask` get installed? It's not part of cargo by default.

Reply:
[Apart from that, 121bcc5 looks fine to me!]

Reply:
`xtask` is an alias defined in `.cargo/config`: https://github.com/softdevteam/yk/pull/214/files#diff-f6009bd0d260464389ace37ab2f89adae993e1fa4a47f779e4c9859937005cedR2

Reply:
Should we mention that you don't have to install anything to dispel any potential confusion?

Reply:
OK, so the answer to my question is that `cargo run --package` installs xtask. We don't need to document that -- I just worried that you guys had installed xtask and forgotten that you'd done so!

Reply:
> cargo run --package installs xtask

It runs the `xtask` package from our workspace, but nothing is installed. Essentially, [this code is executed](https://github.com/vext01/yk_pv/blob/121bcc5691b0fd20d0e7c4cdc6faf3222a1c868e/xtask/src/main.rs#L13).